docs/learn/learn-DOT-inflation.md

```diff
@@ -0,0 +1,102 @@
+---
+id: learn-DOT-inflation
+title: DOT Inflation
```

Comment:
As this doc is conditionally rendered, perhaps change the title to Inflation Rate or something similar?